src/lib/components/Wizard/Wizard.scss

```diff
@@ -28,5 +28,9 @@
     .wizard-form__hidden-content-item {
       display: none;
     }
+    
+    .wizard-form__visible-content-item{
+      height: 100%;
+    }
```

Comment:
use formatting before commit. There should be a space before `{`

Reply:
@mavdryk done.